### PR TITLE
Make GRE parsing work with transparent type

### PR DESCRIPTION
**Pull request by 3ddi**

### CODE REVIEW

File: dpkt/gre.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 GRE_SP = 0x1000  # Sequence Present
 GRE_SS = 0x0800  # Strict Source Route
 GRE_AP = 0x0080  # Acknowledgment Present
+GRE_TRANSPARENT_ETH_BR = 0x6558
```
Comment (obormot) [inline dpkt/gre.py]: Add this to ethernet.py types instead, e.g.
```
ETH_TYPE_TEB = 0x6558  # Transparent Ethernet Bridging
```

**Comment by dimbleby** [comment]
This fix is just what I need!  Is it blocked on something?  Is there anything I can do to help it progress?

**Comment by obormot** [comment]
@dimbleby how about helping to test it? I will take care of the merge conflict

**Comment by dimbleby** [comment]
What did you have in mind?

Can you point me at some existing testcases that I can copy-with-tweaks?  I'd be up for that, I expect.

**Comment by obormot** [comment]
@dimbleby I've just merged the changes to the master branch (via another pull request), please check it it works for you!

**Comment by dimbleby** [comment]
Brilliant, thanks!

**Pull request closed without merging**